code/game/objects/items/weapons/twohanded.dm

```diff
@@ -83,7 +83,7 @@
 /obj/item/weapon/fireaxe/afterattack(atom/A as mob|obj|turf|area, mob/user as mob, proximity)
 	if(!proximity) return
 	..()
-	if(A && wielded && (istype(A,/obj/structure/window) || istype(A,/obj/structure/grille))) //destroys windows and grilles in one hit
```

Comment:
I added it to the grille's attackby(), so there is no feature loss. Did that because otherwise you can destroy invulnerable grilles with fireaxes